docs/Makefile

```diff
@@ -0,0 +1,31 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SPHINXPROJ    = kano-feedback
+SOURCEDIR     = source
+BUILDDIR      = build
+
+
+.PHONY: all clean help Makefile
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

Comment:
Does this need to be in build-depends?

Reply:
Given that it's the `help` target, I'd say it's not necessary. It could be prepended with `-` just in case if you think that's worth it?